packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -149,15 +149,15 @@ export const Product = ({
     accessToken,
     config,
     locationReload,
+    navigateToUrl,
     queryParams,
     matchMediaDefault,
   } = useContext(AppContext);
-  const navigate = useNavigate();
```

Comment:
nit: do we still use the `useNavigate` import on L2?

Reply:
Good catch! Noticed that too and removed in the last commit. 🙂